site/src/pages/index.js

```diff
@@ -1,79 +1,134 @@
-                <li>Paste the code in your GitHub profile README.</li>
-              </ul>
-              </div>
+      <meta
+        name="description"
+        content="Showcasing Your Achievements as a User and a Contributor"
+      />
+      <QueryClientProvider client={queryClient}>
+        <ThemeProvider theme={themeMode}>
+          <GlobalStyle />
+          <Navigation
+            theme={theme}
+            toggleTheme={toggleTheme}
+            showSignUpButton={showSignUpButton}
+            logo={Logo}
+          />
+          <Main>
+            <section className="hero">
+              <img
+                className="program-logo"
+                src={recognitionLogo}
+                alt="Recognition Program Logo"
+              />
+              <h3 className="try-now-txt">
+                Showcasing Your Achievements as a User and a Contributor
+              </h3>
+              <h1>Layer5 Recognition Program</h1>
+
+              <p className="desc-text">
+                Layer5 badges are one of the many ways that we recognize the
+                efforts of our contributors and uplift our users. Layer5 badges
+                represent milestones that you achieve both in using and in
+                contributing to Layer5-supported open source projects. As a
+                contributor, this is how you demonstrate your ownership,
+                dedication, skills and loyalty to Layer5. As a user, this is how
+                you demonstrate your prowess.
+              </p>
+              <div className="banner-main">
+                <img src={recognitionBanner} alt="Recognition Program Banner" />
               </div>
-             <GithubLogo fillColor ={fillColor} width="12rem" height="11rem" />
-            </div>
-              <div className="github-badges">
-                <img src={githubGopi} alt="Recognition Program Banner"/>
-                <img src={githubKaran} alt="Recognition Program Banner"/>
+              <p>
+                See the{' '}
+                <a href="https://layer5.io/community/handbook/recognition">
+                  Recognition
+                </a>{' '}
+                section of the{' '}
+                <a href="https://layer5.io/community/handbook/recognition">
+                  Layer5 Community Handbook
+                </a>{' '}
+                for details of when and how each badge is awarded.
+              </p>
+              <section className="faq">
+                {/* <h1>Frequently Asked Questions</h1> */}
+                <Faq
+                  category={['Layer5 Badges']}
+                  display-category-title={'false'}
+                />
+              </section>
+              <section className="badges">
+                <BadgeGrid />
+              </section>
+              <div className="banner-logos">
+                <h3>
+                  {' '}
+                  Wear your badge pride proudly. Embed your Layer5 badges in
```

Comment:
Please rephrase. Perhaps, just remove “pride”.

Reply:
Yes, I know that you didn’t write this originally, but since you’re changing it, now you get to fix it. 🙂

Reply:
Done